usage/rest-server/src/test/java/brooklyn/rest/resources/ServerResourceTest.java

```diff
@@ -134,4 +131,52 @@ void testGetMissingConfigThrowsException() throws Exception {
             }
         }
     }
+
+    // Alternatively could reuse a blocking location, see brooklyn.entity.basic.SoftwareProcessEntityTest.ReleaseLatchLocation
+    @ImplementedBy(StopLatchEntityImpl.class)
+    public interface StopLatchEntity extends EmptySoftwareProcess {
+        public void unblock();
+        public boolean isBlocked();
+    }
+
+    public static class StopLatchEntityImpl extends EmptySoftwareProcessImpl implements StopLatchEntity {
+        private AtomicBoolean lock = new AtomicBoolean();
+        private boolean isBlocked;
+
+        @Override
+        public void unblock() {
+            synchronized (lock) {
+                lock.set(true);
+                lock.notifyAll();
+            }
+        }
+
+        @Override
+        protected void postStop() {
+            super.preStop();
+            synchronized (lock) {
+                if (!lock.get()) {
```

Comment:
See previous comment in ReleaseLatchLocation.release about `synchronized`. Could change to `CountDownLatch`.